src/shared/services/TopicService.js

```diff
@@ -1,12 +1,14 @@
 import TopicModel from '../models/TopicModel';
-import { db } from './FirebaseService';
+import {db} from './FirebaseService';
+import {reject as filterOut} from 'lodash';
+
 export default class TopicService {
     //TODO: add tests when backend is "added" (called paths)
     //temp
     static people = [
-        { email: 'john@doe.com', name: 'John Doe 1' },
-        { email: 'john_alpha@doe.com', name: 'John Doe 2' },
-        { email: 'john_omega@doe.com', name: 'John Doe 3' },
+        {email: 'john@doe.com', name: 'John Doe 1'},
```

Comment:
ok, as wish, today we're going to be equal 😉

Reply:
https://github.com/warsawjs/topics-manager/pull/27/commits/b0de324e0b6e96429f62828ebc73fa58264f28ae fixed few files in this comment - and removed this piece of code at all :)